pkg/cmd/enclave_setup.go

```diff
@@ -116,17 +116,23 @@ var setupCmd = &cobra.Command{
 				utils.HandleError(errors.New("your project does not have any configs"))
 			}
 
-			var configOptions []string
+			var options []string
+			var defaultOption string
 			for _, val := range configs {
-				configOptions = append(configOptions, val.Name)
+				option := val.Name
+				options = append(options, option)
+				// reselect previously-configured value
+				if val.Name == scopedConfig.EnclaveConfig.Value {
+					defaultOption = val.Name
+				}
```

Comment:
Why should we care about the previous value?

Reply:
So that we can re-select the previous value when re-running setup. I believe this was a feature you requested.

Reply:
Gotcha, can we only re-select the config if the project is the same.

Reply:
I like that

Reply:
Addressed